### PR TITLE
Add Writable Files and Add Chown on Writable Dirs Task

### PR DESCRIPTION
**Pull request by ihsanudin**
Add Writable Files and Add Chown on Writable Dirs Task to support vps based server that do not have acl

### CODE REVIEW

File: recipe/common.php [hunk 1]
```diff
@@ -168,6 +169,17 @@
...
+/**
+ * Make writable files.
+ */
+task('deploy:writable:files', function () {
+    $files = join(' ', get('writable_files'));
+    if (empty($files)) {
+        return;
+    }
+
+    run("$sudo chmod -R 777 $files");
```
Comment (elfet) [inline recipe/common.php]: `$sudo` does not defined.

**Comment by elfet** [comment]
@ihsanudin do you tests your PR?

File: recipe/common.php (hunk 1 above)
Comment (ihsanudin) [inline recipe/common.php]: Sorry @elfet : i just copy that task from recent discussions

**Comment by ihsanudin** [comment]
@elfet : I edit the code directly from github editor so i don't test anything. Just make no error from travis

**Comment by elfet** [comment]
@deployphp/maintainer that do you think about this PR, guys?

**Comment by oanhnn** [comment]
I am using `deployer` with Slim framework and CakePHP framework.
When project run, log files and cache file is created by `www-data`, but onwer of directory `logs`, `tmp` or `cache` is deploy-user. 
I was comment in #327 and i don't agree with this PR.
I think `deployer` will support flow of framework and user can custom original recipe for special project or them.

**Comment by elfet** [comment]
Maybe separate `deploy:writable:files` task to custom recipe (3rd party)? And just merge `chmod -R`/`chown` changes?

**Comment by ihsanudin** [comment]
@oanhnn : This is a common task not a specific framework purpose. I just improve the behavior to dealing with vps that have not and can not set an acl

**Comment by elfet** [comment]
I think this PR need to be separated. @ihsanudin can you do it?

**Pull request closed without merging**